<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Книга рецептов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="placeholderText">
         <string>Поиск по названию, ингредиентам...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Все категории</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Первое</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Второе</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Десерт</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Салат</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Напиток</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="timeFilterComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Любое время</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>До 30 минут</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>До 1 часа</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>До 2 часов</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Более 2 часов</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRecipeButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>➕ Добавить рецепт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="favoritesButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>⭐ Избранное</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="recipesListWidget">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="recipeTextBrowser">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Импорт рецептов...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Экспорт рецептов...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>